### DISCUSSION TITLE
Rozrzucanie podów pomiędzy nodami

### DISCUSSION THREAD

**Discussion by PapinPopin** [AzureCommunityPL/group#480] [Pomoc]
Cześć, czy jest możliwość aby aks tworząc pody zawsze tworzył jeden na zwykłych nodach, a kolejne na nodach spotowych?

**Comment by mmisztal1980** [comment]
https://letmegooglethat.com/?q=how+do+I+assign+pods+to+nodes

**Comment by krukowskid** [comment]
Wydaje mi się, że nie ma to żadnego eleganckiego rozwiązania. Jeżeli korzystasz z helma to można poczarować, żeby tworzyło dwa deploymenty gdzie jeden będzie z jedną replika na regular node,  a pozostałe repliki będą na spotowe node. To oczywiście ma swoje wady i może prowadzić do innych problemów. Wszystkie inne rozwiązania, które mi przychodzą do głowy nie zapewniają, że zawsze będzie jeden i tylko jeden pod dostępny na regularnych node.

Można wykorzystać rozwiązania typu netapp spot, ale nie jest to o co pytasz tylko nakładka na spotowe instancje
